src/redux/news/news.sagas.js

```diff
@@ -23,13 +35,27 @@ function* handleNewsLoad() {
   }
 }
 
+function* handleNewsItemLoad({ payload }) {
+  try {
+    yield put(showLoader());
```

Comment:
Simplify action creators `showLoader` and `hideLoader` to `setLoading` that will take `true` or `false` as arguments

Reply:
Fixed!